index.src.html

```diff
@@ -16,6 +16,15 @@
 </pre>
 
 <pre class="anchors">
+spec: CSSOM; urlPrefix: https://drafts.csswg.org/cssom/
```

Comment:
It should not be necessary to add anything to anchors; Bikeshed autolinking should suffice for this, since CSSOM and DOM are already in Bikeshed.